shogun-lib/src/main/java/de/terrestris/shogun/lib/graphql/GraphQLDataFetchers.java

```diff
@@ -19,36 +17,65 @@
     @Autowired
     UserService userService;
 
+    @Autowired
+    FileService fileService;
+
+    @Autowired
+    ImageFileService imageFileService;
+
+    // Application
     public DataFetcher getApplicationById() {
         return dataFetchingEnvironment -> {
             Long applicationId = ((Integer) dataFetchingEnvironment.getArgument("id")).longValue();
             return this.applicationService.findOne(applicationId);
         };
     }
-
     public DataFetcher getAllApplications() {
         return dataFetchingEnvironment -> this.applicationService.findAll();
     }
 
+    // Layer
     public DataFetcher getLayerById() {
         return dataFetchingEnvironment -> {
             Long layerId = ((Integer) dataFetchingEnvironment.getArgument("id")).longValue();
             return this.layerService.findOne(layerId);
         };
     }
-
     public DataFetcher getAllLayers() {
         return dataFetchingEnvironment -> this.layerService.findAll();
     }
 
+    // User
     public DataFetcher getUserById() {
         return dataFetchingEnvironment -> {
             Long userId = ((Integer) dataFetchingEnvironment.getArgument("id")).longValue();
             return this.userService.findOne(userId);
         };
     }
-
     public DataFetcher getAllUsers() {
         return dataFetchingEnvironment -> this.userService.findAll();
     }
+
+    // File
+    public DataFetcher getFileById() {
```

Comment:
The REST controllers for file entities are accepting the UUID of the file instead of the ID. Not blocking, but I think we should harmonize this behaviour.

Reply:
99b46e9